samples/bluetooth/hci_uart/README.rst

```diff
@@ -6,20 +6,88 @@ Bluetooth: HCI UART
-
-
+Expose the Zephyr Bluetooth Controller support over UART to another device/CPU
+using the H:4 HCI transport protocol (requires HW flow control from the UART).
 
 Requirements
 ************
 
-* BlueZ running on the host, or
 * A board with BLE support
 
+Default UART settings
+*********************
+
+By default the controller builds use the following settings:
+
+* Baudrate: 1Mbit/s
+* 8 bits, no parity, 1 stop bit
+* Hardware Flow Control (RTS/CTS) enabled
+
 Building and Running
 ********************
+
 This sample can be found under :file:`samples/bluetooth/hci_uart` in the
-Zephyr tree.
+Zephyr tree, and it is built as a standard Zephyr application.
+
+Using the Controller with QEMU and BlueZ
+****************************************
+
+The instructions below show how to use a Nordic nRF52832 as a Zephyr BLE
+Controller and expose it to Linux's BlueZ. This can be very useful in order to
+test the Zephyr Link Layer with the BlueZ Host, and even use the controller to
+provide a Linux-based machine with a modern BLE 5.0 controller that can be
+used natively or as a Controller for QEMU-based development.
+
+Make sure you have a recent BlueZ version installed by followint the
+instructions in the :ref:`bluetooth_bluez` section.
+
+Using the Controller with QEMU
+==============================
+
+In order to use the HCI UART controller with QEMU you will need to attach it
+to the Linux Host first. To do so simply build the sample and connect the
+UART to the Linux machine, and then attach it with this command:
+
+   .. code-block:: console
+
+      sudo btattach -B /dev/ttyACM0 -S 1000000 -R
+
+.. note::
+   Depending on the serial port you are using you will need to modify the
+   ``/dev/ttyACM0`` string to point to the serial device your Controller is
+   connected to.
+
+.. note::
+   The ``-R`` flag passed to ``btattach`` instructs the kernel to avoid
+   interacting with the Controller and instead just be aware of it in order
+   to proxy it to QEMU later.
+
+If you are running :file:`btmon` you should see a brief log showing how the
+Linux kernel identifies the attached Controller.
+
+Once the Controller is attached follow the instructions in the
+:ref:`bluetooth_qemu` section to use QEMU with it.
+
+Using the Controller with BlueZ
+===============================
+
+In order to use the HCI UART controller with BlueA you will need to attach it
+to the Linux Host first. To do so simply build the sample and connect the
+UART to the Linux machine, and then attach it with this command:
+
+   .. code-block:: console
+
+      sudo btattach -B /dev/ttyACM0 -S 1000000 -P h4
+
+.. note::
+   Depending on the serial port you are using you will need to modify the
+   ``/dev/ttyACM0`` string to point to the serial device your Controller is
+   connected to.
+
+.. note::
+   The ``-P h4`` flag passed to ``btattach`` instructs the kernel to interact
+   with the Controller using the standard HCI:H4 protocol.
```

Comment:
And this note can then be removed if you remove the `-P h4`

Reply:
ok will do